utils/singleton.py

```diff
@@ -19,4 +19,18 @@ def __call__(cls, *args, **kwargs):
 def destroyAllSingletonInstances():
     global _instances
     _instances = {}
-    
\ No newline at end of file
+
+def noSingletonsAround():
+    return len(_instances)==0
+    
+class ShortSingltonLivesUnderTest:

```

Comment:
Is this class used? It looks like the `my_fixture()` in the other file does the same action per stackoverflow?